preview/Win7Msix/Win7MSIXInstaller/InstallUI.cpp

```diff
@@ -439,6 +484,24 @@ BOOL CreateCancelButton(HWND parentHWnd, RECT parentRect) {
 	return TRUE;
 }
 
+BOOL CreateLaunchButton(HWND parentHWnd, RECT parentRect) {

```

Comment:
> { [](start = 57, length = 2)

style-wise let's be consistent on making this on a separate line by itself

Reply:
Moved bracket to new line.